README.md

```diff
@@ -299,8 +301,8 @@ class fourthcoffee(
 }
 ~~~
 
-As you can see, you can mix and match dsc resources with common puppet resources.
-All [puppet metaparameters](https://docs.puppetlabs.com/references/latest/metaparameter.html) should also be supported.
+As you can see, you can mix and match dsc resources with common Puppet resources.
```

Comment:
Well, at least this is one place where we should change this to "Puppet DSC resources"